index.html

```diff
@@ -317,6 +342,20 @@ <h1>Raspberry Pi Pico Pinout</h1>
                 }
             }
         }
+        function reversed_on_change() {
+            console.log(this.checked);
+            if (this.checked) {
+                document.getElementById("underside").style.display = "block";
+                document.getElementById("upside").style.display = "none";
+                document.getElementById("pinout").style.direction = "rtl";
+                document.getElementsByClassName('labels left')[0].style.direction = "ltr";
+            } else {
+                document.getElementById("underside").style.display = "none";
+                document.getElementById("upside").style.display = "block";
+                document.getElementById("pinout").style.direction = "ltr";
+                document.getElementsByClassName('labels left')[0].style.direction = "rtl";   
+            }
+        }
```

Comment:
I think this could be done entirely with a CSS class swap on the parent element. This would let us create out an alternative backside view with static HTML. I'm not sure exactly what we'd do with that yet, but I'm an ardent NoScript user and like to get by with minimal JS where possible.

I'm not sure how ubiquitous support for this is (my JS is 10 years out of date) but it could boil it all down to just:

```javascript
document.getElementById("pinout").classList.toggle("underside")
```

This also returns a bool that can set the checkbox state.

Reply:
Apart from this pedantry on my part, the reversed view is *brilliant*. Thank you.